runbotics/runbotics-orchestrator-ui/src/views/process/ProcessBuildView/Modeler/ConfigureActionPanel/Actions/getVariablesActions.ts

```diff
@@ -1,7 +1,7 @@
 import { translate } from 'src/hooks/useTranslations';
 import { IBpmnAction, Runner } from './types';
 
-const variablesActions: Readonly<Record<string, IBpmnAction>> = {
+const getVariablesActions: () => Record<string, IBpmnAction> = () => ({
```

Comment:
name of the file should not contain `get` prefix, let go with `variables.actions.ts`, same for others